test/helpers.bash

```diff
@@ -66,23 +80,41 @@ function get_container_netns_path() {
 # it joins the test network namespace before it invokes $NETAVARK,
 # which may be 'netavark' or '/some/path/netavark'.
 function run_netavark() {
-    run_in_host_netns $NETAVARK "$@"
+    local first=$1
+    shift 1
+    run_in_host_netns "$first" $NETAVARK "$@"
 }
 
 ################
 #  run_in_container_netns  #  Run args in container netns
 ################
 #
 function run_in_container_netns() {
-    run_helper nsenter -n -t $CONTAINER_NS_PID "$@"
+    if [ "$1" == "1" ]; then
+        shift 1
+        run_helper nsenter -n -t $CONTAINER1_NS_PID "$@"
+    elif [ "$1" == "2" ]; then
+        shift 1
+        run_helper nsenter -n -t $CONTAINER2_NS_PID "$@"
+    else    
+        die
+    fi
```

Comment:
This is not portable at all. Look at the aardvark tests which already handle more than one containerns do this https://github.com/containers/aardvark-dns/blob/main/test/helpers.bash

Reply:
I will modify it to be an array of NS's but the logic of passing an arg for which one still holds true in aardvark